src/default_panels/StyleLayoutPanel.js

```diff
@@ -28,13 +29,16 @@ const StyleLayoutPanel = (props, {localize: _}) => (
       <ColorPicker label={_('Plot Background')} attr="plot_bgcolor" />
       <ColorPicker label={_('Plot Background')} attr="polar.bgcolor" />
       <ColorPicker label={_('Margin Color')} attr="paper_bgcolor" />
-      <Radio
+      <Dropdown
         label="Hover Interaction"
         attr="hovermode"
         options={[
-          {label: _('Enable'), value: 'closest'},
+          {label: _('Closest'), value: 'closest'},
+          {label: _('X Axis'), value: 'x'},
```

Comment:
I'm afraid this change won't be this easy, as not all plots have X and Y axes :)

Reply:
raised issue: https://github.com/plotly/react-chart-editor/issues/576